e2e/tests/overview-page.spec.ts

```diff
@@ -330,4 +330,20 @@ test.describe('web features mapping progress', () => {
     const textLocator = page.getByText(mappingPattern);
     await expect(textLocator).toBeVisible({visible: false});
   });
+
+  test('Show bookmark title and description', async ({page}) => {
```

Comment:
The behavior asserted here would fit nicely in webstatus-overview-content.test.ts. You should add a test there.

We can't measure coverage in the e2e suite. But we can in the unit tests.

Once you add that test there, you may ask the question, should I keep this test?

This is something I need to add clearer guidance on in the developer docs.

We should reserve the e2e tests for:
1. Tests that show some interaction between the frontend and backend, or
2. Tests that we want to take a screenshot
3. Tests that are really hard to unit test

So for this test:
- we should use the recommended [locators](https://playwright.dev/docs/locators#quick-guide) instead of locating things by id. (We need to do this for some of our existing e2e tests too), and/or
- we should get some data from the backend (which you may want to mock) and/or
- take a screenshot

Reply:
Per offline discussions, it will have a unit test instead